toolkit/tools/imagecustomizerapi/dirpermissions.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+	"os"
+)
+
+// The file permissions to set on the directory.
+//
+// Accepted formats:
+//
+// - Octal string (e.g. "660")
+type DirPermissions os.FileMode
```

Comment:
Why not reuse the `FilePermissions` type?

Reply:
I thought to separate the types to not confuse the design since this type is being set to the directory permissions. Would you suggest to reus the `FilePermissions` type instead here?

Reply:
Will remove this and use the `FilePermissions`